turbopack/crates/turbo-tasks-fs/src/watcher.rs

```diff
@@ -113,20 +113,6 @@ impl DiskWatcher {
         dir_path: &Path,
         root_path: &Path,
     ) -> Result<()> {
-        use anyhow::Context;
```

Comment:
You need to keep this line in favor of the module level import. Otherwise this triggers warnings on some OS.